packer/openstack.pkr.hcl

```diff
@@ -118,10 +138,14 @@ source "openstack" "openhpc" {
 # NB: build names, split on "-", are used to determine groups to add build to, so could build for a compute gpu group using e.g. `compute-gpu`.
 build {
   source "source.openstack.openhpc" {
```

Comment:
I'd prefer to put things which are the same in the two build source blocks into the base `source "openstack" "openhpc"` block. I don't think there's any way to pass different values for the same variable for the different builds (i.e. you'd need different variables for the two builds, like with `source_image_name` and `fatimage_source_image_name`), so the way its done currently is just code duplication.

Reply:
Fair - updated.